### ISSUE TITLE
Keynav crashes when already daemonized

### ISSUE THREAD

**Issue by zarelit** [jordansissel/keynav#5]
Keynav doesn't check wheter another instance is already running, crashing when it's asked to be daemonized twice.

> [zarel ~]$ keynav
> Daemonizing now...
> [zarel ~]$ keynav
> X Error of failed request:  BadAccess (attempt to access private resource denied)
>   Major opcode of failed request:  33 (X_GrabKey)
>   Serial number of failed request:  17
>   Current serial number in output stream:  25

**Comment by zyf0330** [comment]
Exactly, some program like clipper binding ctrl+; will cause keynav cannot start.
